internal/manifest-entry-v1.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
where does this json file get used?

Reply:
I just used them as an example to build the dynamic schema building code. I removed them now that they are no longer useful.